sensor_msgs/NavSatFix gps_fix
---
geometry_msgs/Point utm_coord
